README.md

```diff
@@ -80,209 +34,128 @@ After you're finished please remove all the comments and instructions!
 </details>
 
 <details>
-  <summary>Server</summary>
+<summary>Ruby</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="https://www.ruby-lang.org/">Ruby</a></li>
   </ul>
 </details>
 
 <details>
 <summary>Database</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://www.postgresql.org/">Ruby on Rails</a></li>
   </ul>
 </details>
 
-<!-- Features -->
-
-### Key Features <a name="key-features"></a>
-
-> Describe between 1-3 key features of the application.
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo"></a>
+### Key Features <a name="key-features"></a>
 
-> Add a link to your deployed project.
 
-- [Live Demo Link](https://google.com)
+- **Setup React-Redux and Rails as one app**
+- **Use the api endpoint created in the app in the component**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
-In order to run this project you need:
-
-<!--
-Example command:
+In order to run this project you need have installed:
 
-```sh
- gem install rails
-```
- -->
+- node, npm, gem, yarn, git
+- ruby-3.2.1
+- rails-7.0.4.2
+- @reduxjs/toolkit
+- react-redux
+- react-dom
+- react-router-dom
+- @babel
 
 ### Setup
 
 Clone this repository to your desired folder:
 
```

Comment:
- [ ] Kindly add in details the process a user needs to follow to be able to run your project locally. From the project requirements, this app is started with the command  `**(IMPORTANT: start the app with ./bin/dev command)**` which is absent from your ReadMe. Please make your ReadMe very professional

Reply:
Hi @t-yanick . Thank you for reviewing my project. The command `./bin/dev` is there in the usage section. I think this command is used to run the project. thanks

Reply:
Hi @t-yanick . Could you check the changes I made?